server/src/main/java/org/apache/iotdb/db/qp/Planner.java

```diff
@@ -94,7 +100,7 @@ public PhysicalPlan rawDataQueryToPhysicalPlan(List<String> paths, long startTim
 
     //set time filter operator
     FilterOperator filterOp = new FilterOperator(SQLConstant.KW_AND);
-    Path timePath = new Path("time");
+    Path timePath = new Path(TIME);
```

Comment:
BasicOperatorType also has the raw string "time", it's too ugly, you could replace all this "time" by constant TIME